examples/OECC/README.md

```diff
@@ -0,0 +1,143 @@
+        * 4GL Development System
+        * Progress ProdAS for OE
+
+**Notes:**
+* You can use the `OpenEdge-Samples/examples/OECC/config/response_12.8.3.ini` file as a template and specify the control codes in the file.
+* The demo scripts install OpenJDK to `/usr/lib/jvm/jdk` and OpenEdge to `/psc/dlc`.
+
+### Components 
+The infrastructure uses multiple components. To simplify the deployment, it uses a single virtual machine with some components running at the OS level and others running as a container using Docker.
+
+* Ubuntu Virtual Machine latest 22.04 (tested 22.04.4)
+    * OpenEdge Database 12.8.3
+    * PAS for OpenEdge 12.8.3
+    * OpenEdge Command Center agent 1.3.0
+    * Docker (latest, version tested: 26.1.4)
+        * OpenTelemetry Collector 0.53.0
+        * Prometheus (latest, version tested: v2.53.0)
+        * Grafana (latest, version tested: 9.5.20)
+        * MongoDB 4.4-rc-focal
+        * OpenEdge Command Center server 1.3.0
+
+The OpenEdge Database, PAS for OpenEdge and the OpenEdge Command Center agent are installed at the OS level.
+The OpenTelemetry Collector, Prometheus, Grafana, MongoDB, and the OpenEdge Command Center server are deployed using Docker.
+
+The following diagram shows the components use for OpenTelemetry support.
+![Components](./images/diagram.png)
+
+See related diagram at 
+https://www.progress.com/openedge/features/command-center
+
+### Steps
+
+#### Preparing the installation with the prerequisites
+1. Create `/install` directory used for the installation files.
+~~~
+sudo mkdir -p /install
+sudo chown $USER /install
+~~~
+
+2. Copy the OpenEdge media files and the response file to `/install`.
+
+#### Setting up the environment
+
+1. Clone the OpenEdge-Samples repo:
+~~~
+cd
+git clone https://github.com/progress/OpenEdge-Samples.git
+~~~
+
+2. Run `setup.sh` in the `OECC/scripts` directory. Script uses `sudo` to perform administrator tasks.
+~~~
+cd ~/OpenEdge-Samples/examples/OECC
+./scripts/setup.sh
+~~~
+
+#### Testing the environment
+
+1. Test the configuration.
+~~~
+./scripts/test.sh
+~~~
+
+2. Access OpenEdge Command Center via a web browser:
+    * http://&lt;host-machine&gt;:8000
+    * Use admin/admin to login
+
+3. Access the PAS for OpenEdge instance via a web browser:
+    * http://&lt;host-machine&gt;:8810
+    * http://&lt;host-machine&gt;:8810/web/customer.p
+
+4. Access Prometheus via a web browser:
+    * http://&lt;host-machine&gt;:9090
+    * Use DB_ in the Expression field to query for metrics for the OpenEdge database
+    * Use PASOE_ in the Expression field to query for metrics for PAS for OpenEdge
+
+5. Access Grafana via a web browser:
+    * http://&lt;host-machine&gt;:3000
+    * Use admin/admin to login
+
+**Notes:**
+* The setup script, adds `/etc/rc.local` to start the configuration on startup of the machine. You can also use `./scripts/start.sh` and `./scripts/stop.sh` to manually start and stop the configuration.
+
+### Creating a Dasboard in Grafana
```

Comment:
Typo on "Dashboard", but also why? What is the point of the dashboard in Grafana? Maybe each of these sections needs a single sentence to explain why the item is needed. What is it providing as part of the solution?

Let me state this another way as to why I'm confused. I know that OECC is its own product with its own UI, so I'm not sure what this Dashboard is giving us in addition to OECC. Also, is this optional for use or must this be installed for OECC to run? Users should be informed as to what they're installing and why, and whether it's truly part of the solution or something "nice to have".

Reply:
Fixed typo.
Grafana gives you a way to visualize telemetry data from the OpenEdge database and PASOE.
See blog https://www.progress.com/blogs/using-opentelemetry-metrics-support-openedge-azure

The idea is to provide users with a starting point where they can install OpenEdge Command Center and explorer it.
The Grafana component does not need to be used.